cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -926,6 +926,43 @@ action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = Sysmon
 
 
+[Windows - Firewall is Disabled]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 2 * * * *
+description = The alert will be triggered if the Windows Firewall has been disabled from its active state in the past 24 hours. \
+Data Collection - Windows Firewall Status Check Add-on (https://splunkbase.???????)
+dispatch.earliest_time = -24h
+dispatch.latest_time = now
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = index=* sourcetype=WindowsFirewallStatus \
+| dedup 2 host \
+| stats latest(Domain_Profile_Status) as Domain_Profile_Status latest(Private_Profile_Status) as Private_Profile_Status latest(Public_Profile_Status) as Public_Profile_Status earliest(Domain_Profile_Status) as Previous_Domain_Profile_Status earliest(Private_Profile_Status) as Previous_Private_Profile_Status earliest(Public_Profile_Status) as Previous_Public_Profile_Status count as event_count by host \
+| eval changed = if((Previous_Domain_Profile_Status="ON" AND Domain_Profile_Status="OFF") OR (Previous_Private_Profile_Status="ON" AND Private_Profile_Status="OFF") OR (Previous_Public_Profile_Status="ON" AND Public_Profile_Status="OFF"), "Yes", "No") \
+| search (changed = "Yes") OR ((event_count=1) AND (Domain_Profile_Status="OFF" OR Private_Profile_Status="OFF" OR Public_Profile_Status="OFF")) \
+| rename host as Host \
+| table Host Domain_Profile_Status Private_Profile_Status Public_Profile_Status \
+| eval cyences_severity = case(Public_Profile_Status="OFF" AND Private_Profile_Status="OFF" AND Domain_Profile_Status="OFF", "critical", Public_Profile_Status="OFF", "high", true(), "medium") \
```

Comment:
use simple if condition instead of case

If all profile are off use "high" severity or use "medium"

Reply:
Done